### DISCUSSION TITLE
unable to receive email sent to alias

### DISCUSSION THREAD

**Discussion by r-flo-r** [simple-login/app#1997] [Q&A]
Hello,
i have a problem with receiving emails sent to one of my simplelogin email alias. 
Whats curious: some emails can be received, but others not. It seems it's related to the senderadress of the email. For example, if I send an email from my private email address (the one which i use as a mailbox adress within simple login and the one which i am registered on simplelogin), I am also unable to receive that email. 

I don't know if it contributes to the issue, but when using my home network and trying to access my account on simplelogin.io, i occasionally got an error message:
`Error: Network Timeout
An error occurred while connecting to app.simplelogin.io.`
(Firefox Browser)

After receiving this error on my PC, i also get an error trying to access simplelogin via the android app on my phone (within the same network):
`unknown error: timeout`

if i then quickly switch to mobile connection and retry, the simple login can load the content immediatly, while on PC i got the error.
I am havin these network errors for the simplelogin site since quite a long time. but the issue with receiving messages are just since about two weeks.

Can anyone please help?

Thanks.

**Comment by nguyenkims** [comment]
By default SL doesn't forward emails sent a mailbox to its own alias to avoid creating "loop".

**Reply by r-flo-r** [reply]
i see, but that doesn't explain, why i cant receive emails from the other email adresses.
according to your info, i tried sending an email from one of my other email adresses to the alias with the issue and that worked.

**Comment by r-flo-r** [comment]
tried it with several other adresses, which has worked so far. it seems that its a particular adress from which i cant receive emails via my slmail alias. This particular email adress seems to be: Spotify - no-reply at spotify.com.
why is that? sounds weird, but i dont receive email from this adress.
